notebooks/GitHub_Data_EDA.ipynb

```diff
@@ -0,0 +1,4895 @@
+{
```

Comment:
<p>Small nit, should we clarify here that <code>mi</code> is the tool that does the github data fetching, and <code>mi-scheduler</code> is the tool that sets up scheduled pipelines for data collection? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-services-group/metrics/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>good point! just updated :)</p>